### ISSUE TITLE
OptKing prints Uranium as Silicon

### ISSUE THREAD

**Issue by mdav2** [psi4/psi4#1114]
This input file:
```
memory 5 GB

molecule uf4 {
0 1
U
F 1 1.7
F 1 1.7 2 90.0
F 1 1.7 3 90.0 2 180.0
F 1 1.7 4 90.0 3 180.0
}

set basis LANL2DZ
set scf_type pk
set guess CORE
optimize('hf')
```
produces the following output:
```        
Writing optimization data to binary file.
        Final energy is   -448.6902827985336
        Final (previous) structure:
        Cartesian Geometry (in Angstrom)
           SI     0.0000000000   0.0000000000   0.0000000000
            F     0.0000000000  -1.4770587400   1.4786286784
            F     0.0000000000   1.4770587400   1.4786286784
            F     0.0000000000   1.4770587400  -1.4786286784
            F     0.0000000000  -1.4770587400  -1.4786286784
        Saving final (previous) structure.
        Removing binary optimization data file.
                        --------------------------
                         OPTKING Finished Execution
                        --------------------------

    Final optimized geometry and variables:
    Molecular point group: d2h
    Full point group: D2h

    Geometry (in Angstrom), charge = 0, multiplicity = 1:

    U
    F             1    2.089987
    F             1    2.089987      2   89.939134
    F             1    2.089987      3   90.060866      2 -180.000000
    F             1    2.089987      4   89.939134      3 -180.000000

        Removing binary optimization data file.
        Cleaning optimization helper files.

    Psi4 stopped on: Tuesday, 31 July 2018 04:29PM
    Psi4 wall time for execution: 0:18:57.70

*** Psi4 exiting successfully. Buy a developer a beer!
```

It seems that Psi4 is treating it as Uranium, as the energy is different from SiF4 at the same geometry, and is therefore just a printing problem.

**Comment by JonathonMisiewicz** [comment]
Optking's powers of alchemy are [known](https://github.com/psi4/psi4/issues/1082). Lori could say more about the specifics of how ECPs are causing trouble.

**Comment by loriab** [comment]
fixed by #1109
